### ISSUE TITLE
Cant get the client to work.

### ISSUE THREAD

**Issue by thrawny** [airbnb/hypernova-node#3]
Keep getting this error when using the client.
```js
Requesting MyComponent.js { name: 'Stranger' }
TypeError: Cannot read property 'error' of null
    at /Users/thrawn/Stuff/js/hypernova/node_modules/hypernova-client/lib/index.js:175:30
    at /Users/thrawn/Stuff/js/hypernova/node_modules/hypernova-client/lib/index.js:13:12
    at Array.reduce (native)
    at reduce (/Users/thrawn/Stuff/js/hypernova/node_modules/hypernova-client/lib/index.js:12:27)
    at /Users/thrawn/Stuff/js/hypernova/node_modules/hypernova-client/lib/index.js:171:24
    at process._tickCallback (internal/process/next_tick.js:103:7)
```
Here is my client code:
```js
const express = require('express');
const Renderer = require('hypernova-client');
const devModePlugin = require('hypernova-client/plugins/devModePlugin');

const app = express();

const renderer = new Renderer({
  url: 'http://localhost:3030/batch',
  plugins: [
    devModePlugin,
  ],
});

app.get('/', (req, res) => {
  const jobs = {
    'MyComponent.js': { name: req.query.name || 'Stranger' }
  };

  renderer.render(jobs).then(html => res.send(html));
});

app.listen(8080, () => console.log('Now listening'));
```

If I do a manual http post to the server it responds with data. Payload:
```json
{
  "MyComponent.js": { "name": "MyComponent.js", "data": { "name": "Stranger" } }
}
```
And the server response:
```json
{
    "success": true,
    "error": null,
    "results": {
        "MyComponent.js": {
            "name": "MyComponent.js",
            "html": "<div data-hypernova-key=\"MyComponentjs\"><div data-reactroot=\"\" data-reactid=\"1\" data-react-checksum=\"-1610017535\">Hello, Stranger!</div></div>\n<script type=\"application/json\" data-hypernova-key=\"MyComponentjs\"><!--{\"name\":\"Stranger\"}--></script>",
            "meta": {},
            "duration": 1.267495,
            "statusCode": 200,
            "success": true,
            "error": null
        }
    }
}
```

**Comment by jaredpalmer** [comment]
+1 thought it was just me

**Comment by ljharb** [comment]
v1.0.1 is released - v1.0.0 was [missing lib output](https://github.com/airbnb/hypernova-node/commit/a2bbe4c622119a60437b9fdae97099651722f624). Can you try now?
